torch/csrc/distributed/rpc/rref.h

```diff
@@ -302,6 +307,7 @@ class OwnerRRef final : public RRef {
   c10::optional<T> value_;
   mutable std::mutex mutex_;
   mutable std::condition_variable valueCV_;
+  std::vector<std::shared_ptr<FutureMessage>> futures_;
```

Comment:
As all the futures will be ready when the value is set, I wonder if it would be sufficient to use a single future with a list of callbacks?

Reply:
Oh, right... this is where mapping to folly::Future implicitly in my head (where this wouldn't be possible) is counter-productive. :/ I'll replace with a single FutureMessage.